examples/07_router/src/entries.tsx

```diff
@@ -1,5 +1,6 @@
 import { lazy } from 'react';
 import { createPages } from 'waku';
+import type { PathsForRouter } from 'waku/router';
```

Comment:
Then, this becomes `PathsForPages` or `PathsForPagesRouter`. In the future there could be, `PathsForFooRouter`.